### PR TITLE
Golang Homework 5.2 - Tests/Bench

### PR DESCRIPTION
**Pull request by stevenstr**
**Home work 5.2 - Task:**
● Implement another string to int converter, like
● func myStrToInt2(s str) (int, error){}
● It should pass same tests as in #1, but must be implemented another way
● Add benchmarks for both myStrToInt2 and myStrToInt.
● Find the fastest func

Folders:
First method from HW1, which use an Atoi: **converter**
github.com\stevenstr\converter2>go test -bench=
PASS
ok      github.com/stevenstr/converter  **0.312s**

Second method, which use a fmt.Sscanf: **converter2** 
github.com\stevenstr\converter2>go test -bench=
PASS
ok      github.com/stevenstr/converter2 **1.250s**

### CODE REVIEW

**Review by burov — APPROVED** [review]
Ok, for me

File: converter/converter_atoi.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import "strconv"
+
+//MyStrToIntAtoi func
+func MyStrToIntAtoi(s string) (int, error) {
+	if v, err := strconv.Atoi(s); err == nil {
+		return v, nil
+	} else {
+		return 0, err
+	}
+}
```
Comment (imatsiash-ib) [inline converter/converter_atoi.go:16]: Please see my suggestion in the previous PR.

File: converter2/converter_sscanf.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import "fmt"
+
+//MyStrToIntSscanf func
+func MyStrToIntSscanf(s string) (int, error) {
+	var a int
+	_, err := fmt.Sscanf(s, "%d", &a)
+	if err == nil {
+		return a, nil
+	}
+	return 0, err
```
Comment (imatsiash-ib) [inline converter2/converter_sscanf.go]: Bad style. Will be better to return an error within the `if` chain. The lase return statement should return valid value without error.

Comment (stevenstr) [reply converter2/converter_sscanf.go]: Check pls, I done it some time ago

**Review by imatsiash-ib — APPROVED** [review]

**Pull request closed without merging**